sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlFlexibleServerServerVersionCapability.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Server version capabilities. </summary>
+    public partial class PostgreSqlFlexibleServerServerVersionCapability : CapabilityBase
+    {
+        /// <summary> The status of the capability. </summary>
+        public new string Status { get; }
```

Comment:
What breaking change are we trying to resolve here?

Reply:
Trying to fix this as the status type has changed and is moved to base class:
![image](https://github.com/Azure/azure-sdk-for-net/assets/61298407/2e0a9aa8-fb8a-42fc-8671-41c5b0adc872)

Reply:
If the property `Status` is the same as the previous version (type, setter), it is not a breaking change in this case.

I see the change we have is that we have a base type now, and an existing type now inherits from it and some properties move to the new base. If the moved properties are the same, this is not a breaking change and we do not need to do anything about it.

I did not see ApiCompat tool reporting this issue. We just need to resolve all the issues ApiCompat is reporting.

Reply:
I have removed all new string Status related changes